CHANGELOG.md

```diff
@@ -93,7 +93,8 @@ Shipping methods can be removed by the user after it has been assigned to a chec
   - Add `CheckoutLine.problems` field
   - Add `CheckoutSettingsInput` to `ChannelCreateInput` & `ChannelUpdateInput`
   - Add `checkoutSettings` field to `Channel`
-
+- Add `externalReference` to `updateWarehouse`. It will allow update warehouse by
+  external reference. - #13342 by @Smit-Parmar
```

Comment:
1. Please rebase with latest `main` branch and move your changes to 3.16 section, as 3.15 has been already released.
2. You don't have to break the lines in `CHANGELOG` file.

Reply:
@zedzior Please correct me If I'm wrong but not seeing any releases of 3.15 in repo.

Reply:
There is an alpha version already being tested, so all the new changes will go to 3.16. https://github.com/saleor/saleor/pull/13696